src/components/GoodList/GoodList.tsx

```diff
@@ -0,0 +1,13 @@
+import { GoodCard } from '../GoodCard/GoodCard';
+
+interface List {
+  goods: string[];
+}
+
+export const GoodList: React.FC<List> = ({ goods }) => (
+  <ul>
+    {goods.map(good => (
+      <GoodCard good={good} key={good} />
```

Comment:
According to the checklist, when rendering a list, you should add a `key` prop. While you have added a key prop here, using the `good` string as a key is not a good practice if `good` is not guaranteed to be unique. If there are duplicates or the `goods` array can change in a way that could change the order of items, this could lead to rendering issues. It's better to use a unique identifier like an id.